### ISSUE TITLE
streamline vignette narrative

### ISSUE THREAD

**Issue by vpnagraj** [reconhub/epicontacts#28]
the current vignette is now consolidated into a single file: https://github.com/Hackout3/epicontacts/blob/master/vignettes/epicontacts.Rmd) as of ec6689a

the content is all there thanks to bertrand and nistara but the formatting / narrative make it hard to follow 

@nistara i've assigned myself to this but feel free to take a look ... note that any changes should be made to the rmd file mentioned above unless we want to split demonstrations of features into individual vignettes

**Comment by thibautjombart** [comment]
+1 to this!

**Comment by nistara** [comment]
Great, thanks! I'll look at it and contribute as needed!

**Comment by tc13** [comment]
@vpnagraj This vignette looks great. With the contacts dataset (CD), does this have to contain 2 columns of IDs (ie from and to) followed by (optional) edge attributes in subsequent columns? If so this could be made more explicit

**Comment by vpnagraj** [comment]
see ac963f9
